CMakeLists.txt

```diff
@@ -241,15 +243,28 @@ if (HDF5)
 endif()  
 
 # Parmetis
-if (METIS)
+if ("metis" IN_LIST GRAPH_PARTITIONING_LIBS)
   find_package(METIS REQUIRED)
   if (NOT METIS_64_BIT_INTEGER)
```

Comment:
Ok, that's fair. But isn't the message a bit misleading still? (i.e. maybe replace it by something like "strongly recommended" instead of "it is not going to work")

Reply:
Yes I agree. I changed the warning to:
Found METIS compiled with IDXTYPEWIDTH = 32. It is strongly recommend to compile METIS with IDXTYPEWIDTH = 64, because otherwise the partitioning of large meshes might fail.